dev/TabView/TabView.cpp

```diff
@@ -233,14 +233,42 @@ void TabView::OnListViewLoaded(const winrt::IInspectable&, const winrt::RoutedEv
         // Now that ListView exists, we can start using its Items collection.
         auto items = TabItems();
         auto numItemsToCopy = static_cast<int>(items.Size());
-        if (auto lvItems = listView.Items())
+        bool itemsSourceDoesNotExists = listView.ItemsSource() == nullptr;
+
+        if (itemsSourceDoesNotExists)
+        {
+            if (auto lvItems = listView.Items())
+            {
+                // copy the list, because clearing lvItems may also clear items
+                winrt::IVector<winrt::IInspectable> item_list{ winrt::single_threaded_vector<winrt::IInspectable>() };
```

Comment:
>item_list [](start = 52, length = 9)

I suggest the name itemList, which more closely matches how we name our locals in this project.